docs/model_developers/project_creation.rst

```diff
@@ -44,12 +44,12 @@ An example ``project.yml`` could be:
   repo:
       - url: https://github.com/inmanta/
         type: git
-      - url: https://pypi.org/simple
-        type: package
```

Comment:
Is this way of working deprecated then? Can it still work in combination with the new behavior? Or is it one or the other?

Reply:
This PR deprecated it: https://github.com/inmanta/inmanta-core/commit/c79cc1e1376a13367fa2a3c1949cd4b864ea7fdd
@Hugo-Inmanta do you know if both can be used together?

Reply:
If I understand this correctly:
- on **master** a warning is raised when using an `url` with type `package` but it can be used in combination with the new behaviour
- on **iso6** we are currently reverting the pip index PR's (see discussion [here](https://inmanta.slack.com/archives/CKRF0C8R3/p1688132643347189))
And a separate documentation PR should be opened for iso6 specifically (see Sander's comment [here](https://inmanta.slack.com/archives/CKRF0C8R3/p1688394987395209?thread_ts=1688394690.058399&cid=CKRF0C8R3)